README.md

```diff
@@ -14,13 +14,27 @@ securitas:
 
 ## Features
 
-- List all your installations and add a panel into Home Assistant.
-- Support Sentinel and add two sensor for each Sentinel in each installation you have. The sensor are temperature and humidity.
-- If the option is set to False, the check_alarm will only check the last status that securitas have in their server instead of checking in the alarm itself. This will decrease the number of request that show in your account. In this is set to true and you arm or disarm the alarm not throught Home Assistant, this will likely show a different state. The default value is True.
-- Added a new configuration panel to change some of the options that you set during the setup. Here is an screenshot.
+- Lists all your installations and add a panel into Home Assistant.
+- Supports Sentinel and adds temperature and humidity sensors for each Sentinel in each installation you have.
+- Supports installations with Perimetral (external) alarms.
+
+## Setup
+![Options](./docs/images/setup.png)
+
+- Enter the username and password for your Securitas account.
+- Use 2FA (default: no). It is not necessary to use two-factor authentication to use this integration, but we're leaving that option in case Securitas changes that in the future.
```

Comment:
Hey @cantupaz I think that the default should be yes, enabled by default. I think all countries are actually enabling 2FA right now. That how it was in the past, and the API stopped working. I don't see Securitas not enabling the 2FA or even disabling this. I don't know where you are getting this information from, but this should be enabled by default.